mulighetsrommet-api/src/main/resources/db/migration/V113__add_column_for_estimert_ventetid_for_tiltaksgjennomforing.sql

```diff
@@ -0,0 +1,2 @@
+alter table tiltaksgjennomforing add column estimert_ventetid_verdi int;
+alter table tiltaksgjennomforing add column estimert_ventetid_enhet text;
```

Comment:
Har vi diskutert å heller lagre som 'interval' f. eks? Å konvertere på vei inn? Da blir det lettere å gjøre data science :)

Reply:
Eller antall dager 🤔 Trenger kanskje ikke microsekunder

Reply:
Jeg er ikke kjent med `interval`. Vil det si at vi har et felt istedenfor to i databasen? Og at vi konverterer fra verdiene i frontend til en streng som kan lagres i db?

Reply:
Ja ett felt, for eksempel konvertere til antall dager (en int). Da må man også konvertere tilbake, og velge en enhet som skal vises i frontend basert på hvor mange dager. Usikker. 

Hadde vært enklere om de bare hadde én enhet i frontend da, må de ha måneder også? Hehe la oss teste det her først kanskje 😅